middlewares/passportStrategies.js

```diff
@@ -57,12 +68,112 @@ passport.use(
         if (!user) {
           return done(null, false, { message: 'user does not exist' });
         }
-        return done(null, { ...user.get(), token: token.token });
+        return done(null, user.get());
       } catch (error) {
         return done(error);
       }
     }
   )
 );
 
+passport.serializeUser((user, callback) => {
+  callback(null, user);
+});
+
+passport.deserializeUser(async (user, callback) => {
+  const foundUser = await User.findOne({ where: { id: user.id } });
+  callback(null, foundUser);
+});
+
+passport.use(
+  new TwitterStrategy(
+    {
+      consumerKey: TWITTER_CONSUMER_KEY,
+      consumerSecret: TWITTER_CONSUMER_SECRET,
+      callbackURL: 'http://localhost:3000/api/v1/users/twitter/callback',
```

Comment:
Declare an environment variable for the base url instead of manually writing http://localhost:3000 because in heroku we shall have a public url, not local